src/Tables/TableCell.tsx

```diff
@@ -15,12 +12,17 @@ export enum ALIGN {
 }
 
 export interface TableCellProps {
-  /** Text or components to be displayed in the cell */
-  children: ReactNode;
-  /** Allows you to pass in a alignment property from the ALIGN enum */
+  /**
+   * Allows you to pass in a alignment property from the ALIGN enum.
+   * Defaults to LEFT
```

Comment:
Apparently the `[[ value ]]` style doesn't work in styleguidist. It's using react-docgen under the hood, so maybe that just hasn't been implementing. Too bad, because that would actually be useful.

Reply:
Yeah, that's a shame.